libs/sdk-ui-kit/src/Dialog/ShareDialog/ShareDialogBase/SharedObjectUnderLenientControl.tsx

```diff
@@ -11,16 +12,17 @@ import { ISharedObjectUnderLenientControlProps } from "./types";
 export const SharedObjectUnderLenientControl: React.FC<ISharedObjectUnderLenientControlProps> = ({
     isUnderLenientControl,
     isLeniencyControlSupported,
-    labels,
     onUnderLenientControlChange,
 }) => {
+    const labels = useComponentLabelsContext();
+
     return (
         <SharedObjectCheckboxControl
             isChecked={isUnderLenientControl}
             isSupported={isLeniencyControlSupported}
             onChange={onUnderLenientControlChange}
-            label={labels.underLenientControl}
-            name="shared-object-under-lenient-control"
+            label={labels.accessRegimeLabel}
+            name="shared-oaccessRegimeLabel-control"
```

Comment:
This name change is typo? `oaccessRegimeLabel`? That `o` should probably not be there.

Reply:
done